sktime/clustering/k_means/_k_means_tslearn.py

```diff
@@ -204,8 +204,20 @@ def get_test_params(cls, parameter_set="default"):
             "random_state": None,
             "init": "random",
         }
+        params3 = {
+            "n_clusters": 5,
+            "max_iter": 100,
+            "tol": 1e-5,
+            "n_init": 5,
+            "metric": "euclidean",
+            "max_iter_barycenter": 20,
+            "verbose": 1,
+            "random_state": 42,
+            "init": "random",
```

Comment:
It'd be nice if you can make kmeans++ from first parameter set working.

Reply:
Well, I've stared working on it. Thankyou for your feedback